### PR TITLE
IE8 doesn't support `bind` natively so shim it

### PR DESCRIPTION
**Pull request by tombye**
The JS for selection-buttons uses `Object.prototype.bind` which is supported in Internet Explorer from version 9 upwards. Because of this, whenever we use it, we need to include the shim for `bind` which adds support for <=IE8.

It's annoying debugging toolkit patterns in IE at the moment because you get JS errors thanks to the above. This fixes it.

### CODE REVIEW

File: pages_builder/assets/javascripts/vendor/bind.js [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+                 : oThis,
+                 aArgs.concat(Array.prototype.slice.call(arguments)));
+        };
+
+    fNOP.prototype = this.prototype;
+    fBound.prototype = new fNOP();
+
+    return fBound;
+  };
+}
```
Comment (carolinegreen) [inline pages_builder/assets/javascripts/vendor/bind.js]: Might want to add a newline as we seem to be following this convention in most of the other toolkit javascript files, but apart from that 👍

Comment (allait) [inline pages_builder/assets/javascripts/vendor/bind.js]: I think 4th wall doesn't recognise "👍" in the diff comments, only on the PR itself

Comment (carolinegreen) [inline pages_builder/assets/javascripts/vendor/bind.js]: Yeah, it's a shame.

Comment (tombye) [inline pages_builder/assets/javascripts/vendor/bind.js]: Good catch, it seems to be gone walkies when I copied it across.

**Pull request merged**